connector/signaltometricsconnector/testdata/logs/exponential_histograms/config.yaml

```diff
@@ -0,0 +1,21 @@
+signaltometrics:
+  logs:
+    - name: total.logrecords.exphistogram
+      description: Logrecords as exponential histogram with log.duration from attributes
+      exponential_histogram:
+        count: Int(1)
+        value: Double(attributes["log.duration"])
```

Comment:
Is the Double converter required, or can you also provide `attributes["log.duration"]`?

Reply:
Not required, will simplify.